InterneTaakAfhandeling.Web.Server/Features/User/UserController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Security.Claims;
+using Duende.IdentityModel;
+using InterneTaakAfhandeling.Common.Services.OpenKlantApi.Models;
+using InterneTaakAfhandeling.Web.Server.Authentication;
+using InterneTaakAfhandeling.Web.Server.Middleware;
+using InterneTaakAfhandeling.Web.Server.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InterneTaakAfhandeling.Web.Server.Features.User
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    public class UserController(IUserService userService) : Controller
+    {
+        private readonly IUserService _userService = userService ?? throw new ArgumentNullException(nameof(userService));
+
+        [ProducesResponseType(typeof(List<Internetaken>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ConflictException), StatusCodes.Status409Conflict)]
```

Comment:
I think the body of the response is ProblemDetails  in this case, correct?
```suggestion
        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status409Conflict)]
```